# Request
string map_name

---
# Response
bool   success
string message